docs/declarative-customization/column-formatting.md

```diff
@@ -1688,12 +1688,53 @@ The following example shows how a hyperlink field might be used on a current fie
 }
 ```
 
+**Image fields**
+
+The image field object has the following properties (with example values):
+
+```JSON
+{
+  "fileName": "image.png",
+  "id": "6bb1d843-0633-4c9a-9a16-90bc5abd1d8e",
+  "serverRelativeUrl": "/teams/Discovery/SiteAssets/Lists/ad6ed939-0db2-4d85-8a39-8f3497f41eee/image.png",
+  "serverUrl": "https://contoso.sharepoint.com"
+}
+
+```
+
+<br/>
+
+The following example shows how a image field might be used on a current field.
```

Comment:
The following example shows how an image field can be used on a current field.

Reply:
Fixed!

Reply:
good to go ahead